Basic-Car-Maintenance/Shared/Settings/Tips/ContributionTip.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  ContributionTip.swift
+//  Basic-Car-Maintenance
+//
+//  Created by Sandro Dahl on 24.10.23.
+//
+
+import TipKit
+
+struct ContributionTip: Tip {
+    var title: Text {
+        Text("Thanks for using this app!", comment: "Thanks a user for using the app.")
+    }
+
+    var message: Text? {
```

Comment:
Why is message and image optional, when they're both computed properties that return a value?

Reply:
Otherwise nothing is displayed. Seems to be a TipKit bug.